server/app/assets/javascripts/sort_selector.ts

```diff
@@ -0,0 +1,50 @@
+/** Responsible for controlling the sorting mechanism */
+export function sortSelectorElements(
+  sortSelectId: string,
+  sortElementId: string,
+  sortSublistId: string,
+): void {
+  const listToBeSorted = document.getElementById(
+    sortSelectId,
+  ) as HTMLSelectElement
+  const sublists = document.querySelectorAll(sortSublistId)
+  if (!listToBeSorted || !sublists) {
+    return
+  }
+
+  sublists.forEach((sublist) => {
+    const el: HTMLElement[] = Array.from(
+      sublist.querySelectorAll(sortElementId),
+    )
+
+    const sortedElements = el.sort((elementA, elementB) => {
```

Comment:
Consider defining the comparator as a separate entity? (Actually there seem to be two nested comparators, so two separate entities?) It's quite long to have inlined here. I realize this is technically out of scope but with it living in its own file it seems more logical to do now.

Reply:
I don't mind it the inlining because the code is straightfoward and readable and extracting the compartor functionality to a separate entity feels like it would  be more worth it if we had need of the comparator elsewhere and not just in this function. If you feel strongly about separating the comparators I can do that.

Reply:
👍